lab_3_keywords_textrank/main.py

```diff
@@ -251,7 +294,11 @@ def is_incidental(self, vertex1: int, vertex2: int) -> int:
                 1 if vertices are incidental, otherwise 0
         If either of vertices is not present in the graph, -1 is returned
         """
-        pass
+        if vertex1 not in self._vertices or vertex2 not in self._vertices:
+            return -1
+        idx1 = self._vertices.index(vertex1)
+        idx2 = self._vertices.index(vertex2)
+        return int(self._matrix[idx1][idx2] == 1)
```

Comment:
Could you explain why do you convert integer into boolean and boolean into integer instead of directrly returning the value?